src/pdfmm/private/JpegCommon.cpp

```diff
@@ -1,3 +1,5 @@
+#ifdef PDFMM_HAVE_JPEG_LIB
```

Comment:
Please, don't add such ifdefs to disable whole files but check the source exclusions in CMakeLists for correct working:

[if(NOT PDFMM_HAVE_JPEG_LIB)](https://github.com/pdfmm/pdfmm/blob/27fa0d84d660cb85ec8b7459d96112f3c2e555c0/src/pdfmm/private/CMakeLists.txt#L6)

After you have checked this please resend without the ifdefs in `JpegCommon.h`, `JpegCommon.cpp` .

Reply:
you are right, now it's clear!